pkg/volume/util/hostutil/fake_hostutil.go

```diff
@@ -14,23 +14,35 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package mount
+package hostutil
 
 import (
 	"errors"
 	"os"
 	"sync"
+
+	"k8s.io/kubernetes/pkg/util/mount"
 )
 
-// FakeHostUtil is a fake mount.HostUtils implementation for testing
+// FakeHostUtil is a fake Interface implementation for testing
 type FakeHostUtil struct {
-	MountPoints []MountPoint
+	MountPoints []mount.MountPoint
 	Filesystem  map[string]FileType
 
 	mutex sync.Mutex
 }
 
-var _ HostUtils = &FakeHostUtil{}
+// NewFakeHostUtil returns a struct that implements the HostUtils interface
+// for testing
+func NewFakeHostUtil(mps []mount.MountPoint, fs map[string]FileType) *FakeHostUtil {
```

Comment:
This is true! I'm going to remove that part of the constructor.
I also am adding a comment/TODO to see if the MountPoints are necessary within the FakeHostUtil implementation at all since no callers are setting them.